### PR TITLE
USB: Add keyboardmania

### PR DESCRIPTION
**Pull request by jackun**
~~Untested by me. Missing linux support.~~ Original PR https://github.com/jackun/USBqemu-wheel/pull/60

Signed-off-by: jackun <jack.un@gmail.com>

### CODE REVIEW

File: pcsx2/USB/usb-pad/usb-pad.h [hunk 1]
```diff
@@ -435,16 +455,16 @@ namespace usb_pad
...
 #define GENERIC_PID 0xC294 //actually Driving Force aka PID that most logitech wheels initially report
 #define PID_DF 0xC294
 #define PID_DFP 0xC298 //SELECT + R3 + RIGHT SHIFT PADDLE (R1) ???
 #define PID_DFGT 0xC29A
 #define PID_FORMULA 0xC202 //Yellow Wingman Formula
-#define PID_FGP 0xC20E     //Formula GP (maybe GT FORCE LPRC-1000)
-#define PID_FFGP 0xC293    // Formula Force GP
-#define PID_GTF 0xC293     // as is Formula Force GP
-#define PID_G25 0xC299     // OutRun 2 (jp) supports it apparently
+#define PID_FGP 0xC20E //Formula GP (maybe GT FORCE LPRC-1000)
```
Comment (jackun) [inline pcsx2/USB/usb-pad/usb-pad.h:464]: Keep these clang-format changes?

Comment (lightningterror) [reply pcsx2/USB/usb-pad/usb-pad.h:464]: You can keep them, since not all the comments are aligned, or you can align them all.

**Comment by jackun** [comment]
Yes/no to paned dialogs?

**Comment by RedDevilus** [comment]
Status?

**Comment by jackun** [comment]
I'm trying to generalize evdev portion a little more but eff it, pushing just keyboardmania parts.

**Comment by GovanifY** [comment]
@jackun is this good to go?

**Review by GovanifY — APPROVED** [review]

**Comment by RedDevilus** [comment]
Could you rebase please?

**Pull request merged**